bin/jobsub_submit

```diff
@@ -163,6 +164,29 @@ def do_dataset_defaults(varg: Dict[str, Any]) -> None:
         varg["environment"].append(f"SAM_GROUP={experiment}")
 
 
+def transfer_sandbox(src_dir, dest_url):
+    """Transfer files from src_dir to sandbox with fake_ifdh (gfal-copy).
+    Nothing failing here is considered fatal, since it doesn't affect the job
+    itself, just log availability.
+
+    """
+    print("Transferring files to web sandbox...")
+    try:
+        mkdir_p(dest_url)
+    except Exception as e:
+        print(
+            f"warning: error creating sandbox, web logs will not be available for this submission: {e}"
+        )
+        return
+    for f in os.listdir(src_dir):
+        try:
+            cp(os.path.join(src_dir, f), dest_url)
```

Comment:
Could we make this `cp(os.path.join(src_dir, f), os.path.join(dest_url,f))  ` so we aren't relying on the check if the destintion is a directory behavior?

Reply:
Good call. changed.